submissions/AndriiZahumnyi/js-dom/css/main_style.css

```diff
@@ -0,0 +1,402 @@
+html, body {
+  height: 100%;
+}
+
+body {
+  margin: 0;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  color: rgb(173, 189, 196);
+  background-color: #552586;
+}
+
+button::-moz-focus-inner,
+input::-moz-focus-inner {
+  border: 0;
+  padding: 0;
+}
+
+ul, li, h3 {
+  margin: 0;
+  padding: 0;
+}
+
+h1 {
+  font-size: 4em;
+}
+
+h2 {
+  font-size: 2.5em;
+}
+
+.container {
+  width: 1440px;
+  max-width: 90%;
+  margin: 0 auto;
+  background-color: #552586;
+  height: 100%;
+}
+
+li {
+  list-style-type: none; 
+}
+
+.menu-toggle {
+  display: block;
+  position: absolute;
+  z-index: -1;
+}
+
+                                                  /* Header */
+
+.header {
+  text-align: center;
+  padding: 2em;
+  margin-bottom: 5px;
+}
+
+.logo-wide {
+  display: block;
+}
+
+.logo-thin {
+  display: none;
+}
+
+/* #menu-toggle {display: none;} */
+    
+/* .label-toggle {display: none;} */
+
+                                                  /* Main */
+
+
+
+.main {
+  width: 100%;
+  display: flex;
+  margin-top: 5px;
+}
+
+                                              /* Aside menu */
```

Comment:
You can remove these comments as well. it's quite easy to follow your styles, so no need to add comments